zipline/assets/_assets.pyx

```diff
@@ -251,6 +251,21 @@ cdef class Future(Asset):
         if self.end_date is None:
             self.end_date = expiration_date
 
+    property auto_close_date:
```

Comment:
Yep - Andrew and Stewart suggested the same thing, and I'm leaning toward it, too. It has a bigger impact on look-up in the AssetFinder, but that is a surmountable issue.